packages/atomic/cypress/e2e/external.cypress.ts

```diff
@@ -1,6 +1,7 @@
 import {RouteAlias, setupIntercept} from '../fixtures/fixture-common';
 
-describe('External Test Suite', () => {
+// TODO: https://coveord.atlassian.net/browse/KIT-3540 - rewrite in playwright
+describe.skip('External Test Suite', () => {
```

Comment:
What's the error that happens on those?

Reply:
<img width="954" alt="image" src="https://github.com/user-attachments/assets/2a65b8e0-9d56-45b4-bf8d-83107b177443">

Reply:
Probably something to do with the cypress setup for this file in particular. It is a special case from usual cypress tests.